delta/utils/register.py

```diff
@@ -82,94 +82,159 @@ def __init__(self):
   serving = Register('serving')
 
 
+NLP_TASK_MODULES = [
```

Comment:
NLP_TASK_MODULES + SPEECH_TASK_MODULES = ALL

Reply:
Some modules may be both used in nlp and speech.